<launch>

  <!--node name="map_server" pkg="map_server" type="map_server" args="$(find my_2d_nav)/maps/playground.pgm"/-->
  <!--- Run AMCL --> 
      <!--include file="$(find my_2d_nav)/launch/amcl_diff.launch"/-->
      


  <node pkg="move_base" type="move_base" respawn="false" name="move_base" output="screen">
    <remap from="cmd_vel" to="mobile_base/commands/velocity"/>
    <param name="base_global_planner" value="global_planner/GlobalPlanner"/>
    <param name="base_local_planner" value="dwa_local_planner/DWAPlannerROS"/>
    <!--param name="base_local_planner" value="teb_local_planner/TebLocalPlannerROS"/-->
    <!--param name="base_local_planner" value="base_local_planner/TrajectoryPlannerROS"/-->
    <rosparam file="$(find my_2d_nav)/move_base_config/costmap_common_params.yaml" command="load" ns="global_costmap" />
    <rosparam file="$(find my_2d_nav)/move_base_config/costmap_common_params.yaml" command="load" ns="local_costmap" />
    <rosparam file="$(find my_2d_nav)/move_base_config/local_costmap_params.yaml" command="load" />
    <rosparam file="$(find my_2d_nav)/move_base_config/global_costmap_params.yaml" command="load" />
    <rosparam file="$(find my_2d_nav)/move_base_config/move_base_params.yaml" command="load" />
    <rosparam file="$(find my_2d_nav)/move_base_config/global_planner_params.yaml" command="load"/>
    <!--rosparam file="$(find my_2d_nav)/move_base_config/base_local_planner_params.yaml" command="load"/-->
    <rosparam file="$(find my_2d_nav)/move_base_config/dwa_local_planner_params.yaml" command="load"/>
    <!--rosparam file="$(find my_2d_nav)/move_base_config/teb_local_planner_params.yaml" command="load"/-->

  </node>


</launch>
